### DISCUSSION TITLE
Is there any way to parse codeforce problem and submit it via FastOlympicCoding?

### DISCUSSION THREAD

**Discussion by dhimanda** [Jatana/FastOlympicCoding#38] [Q&A] [answered]
I am huge fan of fastOlympicCoding.  Thanks a lot to Jatana for making this. 
I use competitive companion + epEditor + cfTool for parse and submit code to codeforces directly. 

I will loved to if there any way to do this with SublimeText by help of fastOlympicCoding.  

THANKS

**Comment by dhimanda** [comment, marked as answer]
I found [FastOlympicCodingHook](https://github.com/DrSwad/FastOlympicCodingHook) by @DrSwad  which helps me to parse test case from many OJ via Competitive Companion chrome extension. 

Thanks to the developer and of course to  @Jatana.

**Reply by Gr-Nobel** [reply]
Competitive programming phraser has more fuctionality. just can create files in specified directory.
